package/base-files/files/lib/functions.sh

```diff
@@ -92,7 +92,7 @@ config_unset() {
 # config_get <section> <option>
 config_get() {
 	case "$3" in
-		"") eval echo "\${CONFIG_${1}_${2}:-\${4}}";;
+		"") eval echo "\"\${CONFIG_${1}_${2}:-\${4}}\"";;
 		*)  eval export ${NO_EXPORT:+-n} -- "${1}=\${CONFIG_${2}_${3}:-\${4}}";;
```

Comment:
Doesn't this line have the quoting issue as well?

Reply:
The "eval export" line? No, it doesn't. The quoting issue comes from using echo and eval.

Reply:
You may want to test it with an option set to '*'.

Reply:
I think the proper explanation why the eval export line doesn't have the issue is that a plain (export) variable assignment is not a command that's subject to a new level of expansion after eval, but echo is one.